ocaml/network/network_utils.ml

```diff
@@ -751,8 +752,14 @@ module Ovs = struct
 			else
 				[]
 		in
+		let set_mac_table_size =
+			if vlan = None then
+				["--"; "set"; "bridge"; name; "other_config:mac-table-size=" ^ (string_of_int !mac_table_size)]
+			else
+				[]
+		in
 		vsctl ~log:true (del_old_arg @ ["--"; "--may-exist"; "add-br"; name] @
```

Comment:
If this is not just a backport, it would be better (more efficient, cleaner) to use `List.concat` here.